docs/theme-specification.md

```diff
@@ -7,49 +7,88 @@ The object itself is not coupled to Styled System's implementation and can be us
 ## Scale Objects
 
 Many CSS style properties accept open-ended values like lengths, colors, and font names.
-In order to create a consistent styling system, the theme object is centered around the idea of scales, such as a typographic (font-size) scale, a spacing scale for margin and padding, and a color object.
-These scales can be defined in multiple ways depending on needs, but tend to use arrays for ordinal values like font sizes, or plain objects for named values like colors, with the option of nesting objects for more complex systems.
+In order to create a consistent styling system, the theme object is centered around the idea of scales, such as a typographic (font-size) scale, a spacing scale for margin and padding, and a color object. Scales must be JSON serializable - this way can be used in JS and CSS but possibly even cross platform.
```

Comment:
I really like the idea of being JSON serializable – is this also part of the reason you changed the approach to aliasing?

Reply:
yeah, I was trying to also to come up with a way to make it easy for tools to make look ups

Reply:
Yeah, ensuring that scales are JSON serializable is 💯. Love that idea.